packages/react/src/TransNoContext.tsx

```diff
@@ -54,7 +54,7 @@ export function TransNoContext(
 
   const { values, components } = getInterpolationValuesAndComponents(props)
 
-  const _translation: string =
```

Comment:
having this explicitly typed makes it more readable imho. With the type, you and the compiler know what it should be.

Can't say that about

```
const _translation =
    i18n && typeof i18n._ === "function"
      ? i18n._(id, values, { message, formats })
      : id
```